packages/vscode-extension/src/extension.ts

```diff
@@ -623,6 +624,17 @@ async function initializeContextKey() {
     workspaceUri && (await isM365Project(workspaceUri.fsPath))
   );
 
+  const workspacePath = workspaceUri?.fsPath;
+  if (workspacePath) {
```

Comment:
Is this intended to check whether it's teamsfx project? If so, you could use `isTeamsFxProject`.

Reply:
Yes, you're right, will updatge